src/styles/_class-overrides.scss

```diff
@@ -0,0 +1,525 @@
+  .banner__language-selector-item--LtoR {
+    @include hover-border(
+      'left',
+      'primary',
+      'accent-cool-lighter',
+      'base-dark'
+    );
+  }
+  //all language buttons when there are 5 or fewer languages
+  .banner__language-selector-button-group {
+    @include flex-center-space-between();
+    .banner__language-selector-item {
+      @include hover-border(
+        'top',
+        'primary',
+        'accent-cool-lighter',
+        'base-dark'
+      );
+      margin: 0 units('105');
+      &:hover {
+        padding-top: 0;
+      }
+    }
+  }
+}
+
+//circular images in graphics section
+.graphic_image--circle {
+  border-radius: 50%;
+}
+
+//mobile & tablet view
+@media (min-width: units('mobile')) {
+  //hide language selector button and project title
+  .banner__language-selector,
+  .header__title {
+    display: none;
+  }
+  .usa-nav {
+    padding: 0;
+    .usa-nav__inner {
+      //nav close button
+      .usa-nav__close {
+        margin: units(1);
+        padding: units(1);
+      }
+      .usa-nav__primary-item {
+        border: none;
+        //these submenu items are shown when a nav label is clicked and the accordion button expands
+        .usa-nav__submenu {
+          @include u-bg('accent-cool-lighter');
+          .usa-nav__submenu-item {
+            @include u-text('base-darker');
+            @include u-bg('accent-cool-lighter');
+            border: none;
+          }
+        }
+        //links and nav label buttons in the nav menu
+        //styling applies when accordion buttons are open and closed
+        a,
+        .usa-accordion__button[aria-expanded='false'],
+        .usa-accordion__button[aria-expanded='true'] {
+          @include hover-border(
+            'left',
+            'secondary',
+            'base-darker',
+            'accent-cool-lighter'
+          );
+          @include u-text('base-darker');
+          @include u-bg('accent-cool-lighter');
+          padding: 0 units(0.5);
+          margin: units(1.5) units(2);
+          &:hover {
+            @include u-text('base-darker');
+            @include u-bg('accent-cool-lighter');
+            @include u-text('bold');
+            padding: 0 units(0.5);
+            margin: units(1.5) units(2);
+          }
+        }
+      }
+    }
+  }
+
+  //header in mobile view
+  .usa-navbar {
+    @include flex-center-space-between();
+    margin: units(4) 0 units(2) 0;
+    //smaller logo size
+    .header__logo {
+      width: units('card');
+      height: auto;
+      padding-left: units(2);
+    }
+    //nav menu button
+    .usa-menu-btn {
+      @include u-bg('secondary');
+    }
+  }
+
+  //languages in nav bar
+  .header__language-nav-container {
+    //position languages at the bottom
+    position: absolute;
+    bottom: 0;
+    @include u-bg('base-dark');
+    padding: units(1) 0;
+    //right align right to left languages
+    .header__language-nav-RtoL {
+      display: flex;
```

Comment:
I know you were going to use mixins, but one can be done here.

Reply:
done

Reply:
There are repetitions in lines 403-407 and 411-415, should we leave that as it is?